py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -2,9 +2,11 @@
 atomic_swap_ctrt contains Atomic Swap contract.
 """
 from __future__ import annotations
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Dict, Any, Union
+from py_v_sdk.data_entry import DataStack
```

Comment:
```suggestion
from py_v_sdk import data_entry as de
```
Then call `DataStack` as `de.DataStack`